### PR TITLE
splitted deletion of test concept and table into seperate test cases …

### PR DESCRIPTION
**Pull request by FabianBinder21**
…and added delay

### CODE REVIEW

**Comment by FabianBinder21** [comment]
@thoniTUB we will restart the e2e-tests multiple times to ensure that the fix worked. After that i will assign you as the reviewer.

**Comment by FabianBinder21** [comment]
@thoniTUB I have started the e2e job multiple times without any failures. Our thought process was, that the docker container performance varies and after the page reload triggered by previously deleting the test concept, the test runs before the page reload has finished. Therefore we added a delay

File: cypress/integration/backend-admin-ui/test_2_dataset.js [hunk 1]
```diff
@@ -61,20 +61,21 @@ context("Admin UI Single Dataset", () => {
...
         it("Can delete test concept", () => {
             cy.get('[data-test-id="accordion-Concepts"]').click();
             cy.get(`[data-test-id="delete-btn-concept-${testDSID}.concept1"]`).click({force: true});
             cy.get(`[data-test-id="delete-btn-${testDSID}.concept1"]`).should('not.exist');
         });
+    });
 
+    describe("Can delete test table", () => {
         it("Can delete test table", () => {
+            cy.wait(100);
```
Comment (thoniTUB) [inline cypress/integration/backend-admin-ui/test_2_dataset.js:74]: Does this wait decides whether the test passes or not?
It looks tricky.
If possible i propose to call `visitAdminUI(`datasets/${testDSID}`);` instead

Comment (FabianBinder21) [reply cypress/integration/backend-admin-ui/test_2_dataset.js:74]: It should make a difference. Locally there is no change if the wait is set or not, it always passes.
We thought about using `visitAdminUI` but I was unsure, if it actually reloads the page if the same url is given.
But after reading the documentation (https://docs.cypress.io/api/commands/visit) it seems like it should always reload.
I will change it to `visitAdminUI`.

File: cypress/integration/backend-admin-ui/test_2_dataset.js [hunk 2]
```diff
@@ -61,20 +61,24 @@ context("Admin UI Single Dataset", () => {
...
+        before(() => { visitAdminUI(`datasets/${testDSID}`); });
 
         it("Can delete test concept", () => {
             cy.get('[data-test-id="accordion-Concepts"]').click();
             cy.get(`[data-test-id="delete-btn-concept-${testDSID}.concept1"]`).click({force: true});
             cy.get(`[data-test-id="delete-btn-${testDSID}.concept1"]`).should('not.exist');
         });
+    });
+
+    describe("Can delete test table", () => {
```
Comment (thoniTUB) [inline cypress/integration/backend-admin-ui/test_2_dataset.js:74]: If you know put both `it` back in one `describe` it should have the same effect, doesn't it?

Comment (FabianBinder21) [reply cypress/integration/backend-admin-ui/test_2_dataset.js]: Yes, but only if `visitAdminUI` is called after the test concept got deleted. Therefore i changed `before` to 'beforeEach` to make sure the page does reload before the deletion of the test table and combined both tests under one `describe` again.
If you got nothing more to add i would restart the e2e job multiple times again to make sure it works and merge the pr afterwards.

**Pull request merged**